test/integration/app.spec.ts

```diff
@@ -32,7 +32,7 @@ describe('admin', () => {
     expect(gcloud).to.be.undefined;
   });
 
-  it('calling admin.firestore loads Firestore', () => {
+  it('Loads Firestore when calling admin.firestore', () => {
```

Comment:
Oops, should be lower case: 'loads Firestore...'

Reply:
Done